tool/src/org/antlr/v4/automata/LexerATNFactory.java

```diff
@@ -466,10 +470,10 @@ public IntervalSet getSetFromCharSetLiteral(GrammarAST charSetAST) {
 						state = CharSetParseState.ERROR;
 						break;
 					case CODE_POINT:
-						state = applyPrevStateAndMoveToCodePoint(charSetAST, set, state, escapeParseResult.codePoint);
+						state = applyPrevStateAndMoveToCodePoint(charSetAST, set, state, escapeParseResult.codePoint, caseInsensitive);
```

Comment:
Given `caseInsensitive` is a field, can we remove some of these arguments? At first glance, `applyPrevStateAndMoveToCodePoint()` is a method in same factory class so we could avoid the arg.

Reply:
Yes, we can. I left them for the furture if single-token case insensitivity is implemented in some way. I'll remove for simplicity.